docs/querying/examples.md

```diff
@@ -6,6 +6,11 @@ sort_rank: 4
 
 # Query examples
 
+Most of these examples will work on a standard installation of Prometheus.
+Once these examples start using metric names starting with `instance_`, you
```

Comment:
There are no metrics in these examples starting with instance_. I'd suggest working just off Prometheus metrics, as these docs are tied to the Prometheus version. Also be aware the http metric is different in master.